app/views/case_contacts/_case_contact.html.erb

```diff
@@ -31,12 +31,16 @@
   <td>
     <span class="mobile-label">Creator</span>
     <% if policy(contact).edit? %>
-      <% if contact.creator&.supervisor? %>
-        <%= link_to "#{contact.creator&.display_name} ", edit_supervisor_path(current_user) %>
-      <% elsif contact.creator&.casa_admin? %>
-        <%= link_to "#{contact.creator&.display_name} ", edit_users_path %>
+      <% if current_user.volunteer? %>
+        <%= contact.creator&.display_name %>
```

Comment:
This should actually be using the `name` decorator, which will always exist. So you can change it to:

```
contact.creator.decorate.name
```

Reply:
Just to double check, the decorator added some extra bits of info to the user's name. Is this what it's supposed to look like or have I done something wrong?
<img width="356" alt="Screenshot 2020-11-15 at 08 33 19" src="https://user-images.githubusercontent.com/22390758/99180345-5fd10200-271d-11eb-93f0-788e8d896483.png">

Reply:
decorator name vs display_name is an interesting conversation. I don't think it's an essential distinction here. before names were required, we would default name to email when name was missing. The "org1MM" prefix is from a recent seed data change I made to help us track down multi-casa bugs.